tests/Fixtures/TestBundle/Entity/SoMany.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiFilter;
+use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\OrderFilter;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\RangeFilter;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ApiResource(attributes={
+ *     "pagination_partial"=true,
+ *     "pagination_via_cursor"={
```

Comment:
what about supporting multiple fields? this is often how it is done i postgres...
`SELECT * FROM table WHERE (date, id) < (prev_date, prev_id) ORDER BY date DESC, id DESC LIMIT 10;`

Reply:
I was thinking about adding more later but I think you're right, we can start with multiple.

Reply:
Multiple fields added 👍

Reply:
Rather than just supporting fields, it might be _needed_. Unlike the sorting for a regular pagination, the cursor _must_ be unique. Although not efficient performance-wise, the safest way to achieve this is to always append (when necessary as it requires what fields are already present) a sort by ID as a last order by.

I also think you need to account for the `NULL` values as they might mess up the order